platform_versioned_docs/version-24.1.1/getting-started/quickstart-demo/view-run-information.mdx

```diff
@@ -0,0 +1,141 @@
+
+When you launch a pipeline, you are directed to the **Runs** tab which contains all executed workflows, with your submitted run at the top of the list. 
+
+Each new or resumed run is given a random name, which can be customized prior to launch. Each row corresponds to a specific run. As a job executes, it can transition through the following states:
+
+- **submitted**: Pending execution
+- **running**: Running
+- **succeeded**: Completed successfully
+- **failed**: Successfully executed, where at least one task failed with a terminate error strategy
+- **cancelled**: Stopped forceably during execution
+- **unknown**: Indeterminate status
+
+![View runs](assets/sp-cloud-view-all-runs.gif)
+
+### View run details for nf-core/rnaseq
+
+The pipeline launched [previously](./launch-pipelines.mdx) is listed on the **Runs** tab. Select it from the list to view the run details.
+
+#### Run details page
+
+As the pipeline runs, run details will populate with the following tabs:
+
+- **Command-line**: The Nextflow command invocation used to run the pipeline. This contains details about the pipeline version (`-r 3.14.0` flag) and profile, if specified (`-profile test` flag).
+- **Parameters**: The exact set of parameters used in the execution. This is helpful for reproducing the results of a previous run.
+- **Resolved Nextflow configuration**: The full Nextflow configuration settings used for the run. This includes parameters, but also settings specific to task execution (such as memory, CPUs, and output directory).
+- **Execution Log**: A summarized Nextflow log providing information about the pipeline and the status of the run.
+- **Datasets**: Link to datasets, if any were used in the run.
+- **Reports**: View pipeline outputs directly in the Platform.
+
+![View the nf-core/rnaseq run](assets/sp-cloud-run-info.gif)
+
+### View reports
+
+Most Nextflow pipelines generate reports or output files which are useful to inspect at the end of the pipeline execution. Reports can contain quality control (QC) metrics that are important to assess the integrity of the results.
+
+![Reports tab](assets/reports-tab.png)
+
+For example, for the nf-core/rnaseq pipeline, view the [MultiQC](../../../../multiqc_docs/multiqc_repo/docs/markdown/index.mdx) report generated. MultiQC is a helpful reporting tool to generate aggregate statistics and summaries from bioinformatics tools.
+
+![Reports MultiQC preview](assets/reports-preview.png)
+
+The paths to report files point to a location in cloud storage (in the `outdir` directory specified during launch), but you can view the contents directly and download each file without navigating to the cloud or a remote filesystem.
+
+#### Specify outputs in reports
+
+To customize and instruct Platform where to find reports generated by the pipeline, a [tower.yml](https://github.com/nf-core/rnaseq/blob/master/tower.yml) file that contains the locations of the generated reports must be included in the pipeline repository. 
```

Comment:
That's a good point. We only have docs about the tower.yml file in the Platform Enterprise configuration context, but not the pipeline context. It might be worthwhile to expand the pipeline schema page in the Platform docs to a broader pipeline config page. I'll create a task in the backlog to revisit this.

Reply:
https://seqera.atlassian.net/browse/EDU-259